src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -21,10 +21,15 @@ public class BuildEventArgsWriter
             typeof(LazyFormattedBuildEventArgs).GetField("arguments", BindingFlags.Instance | BindingFlags.NonPublic);
         private static FieldInfo buildEventArgsMessageField =
             typeof(BuildEventArgs).GetField("message", BindingFlags.Instance | BindingFlags.NonPublic);
-        private static Func<CultureInfo, string, object[], string> formatStringDelegate =
-            (Func<CultureInfo, string, object[], string>)Delegate.CreateDelegate(
-                typeof(Func<CultureInfo, string, object[], string>),
-                typeof(LazyFormattedBuildEventArgs).GetMethod("FormatString", BindingFlags.Static | BindingFlags.NonPublic));
+
+        // TODO: Delegate.CreateDelegate will only become available on .NET Standard 2.0, for now just use a MethodInfo
+        //private static Func<CultureInfo, string, object[], string> formatStringDelegate =
+        //    (Func<CultureInfo, string, object[], string>)Delegate.CreateDelegate(
+        //        typeof(Func<CultureInfo, string, object[], string>),
+        //        typeof(LazyFormattedBuildEventArgs).GetMethod("FormatString", BindingFlags.Static | BindingFlags.NonPublic));
+
+        private static MethodInfo formatStringMethodInfo =
+            typeof(LazyFormattedBuildEventArgs).GetMethod("FormatString", BindingFlags.Static | BindingFlags.NonPublic);
```

Comment:
> That's not supported till .NET Standard 2.0 :(

Just FYI, since it's all moot for this change, but MethodInfo.CreateDelegate has been supported replacement for Delegate.CreateDelegate since Windows 8, definitely pre .NET Standard 2.0.  There had been an effort to prevent reflection from being available in every platform, so Delegate could note depend on reflection, but the reverse was fine.